src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,46 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ imgUrl, title, description, imdbUrl }) => (
+
+  <div className="card">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img src={imgUrl} alt="Film logo" />
+      </figure>
+
+      <div className="card-content">
+        <div className="media">
+          <div className="media-left">
+            <figure className="image is-48x48">
+              <img src="/images/imdb-logo.jpeg" alt="imdb" />
```

Comment:
```suggestion
              <img src="./images/imdb-logo.jpeg" alt="imdb" />
```
Зміни шлях бо картинки не загружаються https://prnt.sc/u9m1yh